client/js/actions/view-action-creator.js

```diff
@@ -4,25 +4,39 @@ var ApiUtils = require('../utils/api-utils');
 var AppConstants = require('../constants/constants');
 
 var ViewActionCreator = {
+
   addEvent: function (event) {
     AppDispatcher.dispatch({
       actionType: AppConstants.ViewActionTypes.ADD_EVENT
     });
     ApiUtils.addEvent(event, ServerActionCreator.eventAdded);
   },
+
   getDayData: function(date){
     AppDispatcher.dispatch({
       actionType: AppConstants.ViewActionTypes.GET_DAY_DATA
     });
 
-    ApiUtils.getDayData(date, ServerActionCreator.recieveDayData);
+    ApiUtils.getDayData(new Date('Jun 21 2015'), ServerActionCreator.recieveDayData);
```

Comment:
Thanks for flagging this as hardcoded in the PR, would be great to throw in a comment to the same effect in the file itself